apps/etterlatte-behandling/src/main/kotlin/behandling/klage/KlageService.kt

```diff
@@ -302,6 +312,45 @@ class KlageServiceImpl(
         return klageMedOppdatertResultat
     }
 
+    override fun avbrytKlage(
+        klageId: UUID,
+        aarsak: AarsakTilAvbrytelse,
+        kommentar: String,
+        saksbehandler: Saksbehandler,
+    ) {
+        val klage =
+            hentKlage(klageId)
+                ?: throw NotFoundException("Klage med id=$klageId finnes ikke")
+        if (!KlageStatus.kanAvbryte(klage.status)) {
+            throw IkkeTillattException("KLAGE_KAN_IKKE_AVBRYTES", "Klage med id=$klageId kan ikke avbrytes")
+        }
+
+        val avbruttKlage =
+            klage.copy(
+                status = KlageStatus.AVBRUTT,
+                aarsakTilAvbrytelse = aarsak,
+            )
```

Comment:
```suggestion
        val avbruttKlage = klage.avbryt(aarsak)
```
Og heller ha en medlemsmetode på `Klage`-dataklassen slik som for ferdigstill etc